### ISSUE TITLE
lib exists

### ISSUE THREAD

**Issue by olehmisar** [grjte/noir_webauthn#1]
in case you want to draw inspiration or fork:
https://github.com/olehmisar/noir_webauthn

**Comment by grjte** [comment]
Thanks, that's helpful! I'll finish this one out because it's further along locally & I also want to prove the registration for the next thing I'm building. Are you using your library anywhere yet? (Maybe not since it's 1 hour old lol) I'm interested to see what you're building it for

**Comment by olehmisar** [comment]
Yeah, I am using it here for a passkey account: https://wallet.shieldswap.org.

I am decomposing my private monorepo into open source libs, so that's why it's only 1 hour old.
